pkg/kdump/config-client-suse.js

```diff
@@ -182,7 +182,7 @@ export class ConfigFileSUSE extends ConfigFile {
             settings._internal[key].value = value;
         } else {
             if (value)
-                settings._internal[key] = { value: value };
+                settings._internal[key] = { value };
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)